### PR TITLE
Added application wrapper

### PR DESCRIPTION
**Pull request by trym-b**
Initial Application wrapper added, contains the very basic of C++ Application's
functionality: connect_camera and create_file_camera

Installed pytest-pythonpath for setting the correct pythonpath

Added pytest.ini for setting python path when running pytest (or else it
cannot find wrapper)

Installed pytest-timeout for timing out application tests

Added samples: sample capture, sample capture hdr and sample capture from file
All of these samples will be extended as more wrapper functionality comes alive

Temporarily disabled duplicate pylint warning until samples are beginning
to diverge

Disabled bad-continuation since it is in conflict with black

Unable to run linting in a virutal environment, even if deactivating and
reactiviating after install. Pylint gets import-error in the virtual env,
but this does not happen outside the virtual env

whitelisted zivid package from pylint since it contains c/c++ python code
that pylint struggels with.

Fixture scope should preferably be "function" instead of "session", but
we don't have a "release" function yet so it has to be application

travis file can be mostly reverted

### CODE REVIEW

File: test/test_application.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def test_init_with():
+    import wrapper
+
+    with wrapper.Application() as app:
+        assert app
+
+
+def test_release(application):
```
Comment (nedrebo) [inline test/test_application.py]: This does not test anything. To test if release did work you should probably try to use application afterwards and except an exception.

File: samples/sample_capture_hdr.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import wrapper
```
Comment (nedrebo) [inline samples/sample_capture_hdr.py]: Better to get the naming correct from the beginning. wrapper should be zivid and current zivid should be _zivid or __zivid, or something else if you have better ideas.

File: test/test_application.py (hunk 1 above)
Comment (nedrebo) [reply test/test_application.py]: And probably do the same before as well and expect no exception to check that release changes the state of application.

Comment (trym-b) [reply test/test_application.py]: Then I'll remove it

File: samples/sample_capture_hdr.py (hunk 2 above)
Comment (trym-b) [reply samples/sample_capture_hdr.py]: Not trivial renaming module. PYTHON_MODULE_NAME is not the sole variable for setting the module name.

**Pull request closed without merging**